lightning.install

```diff
@@ -8,6 +8,38 @@
 use Drupal\Core\Entity\Entity\EntityViewDisplay;
 use Drupal\field\Entity\FieldConfig;
 use Drupal\lightning_core\ConfigHelper as Config;
+use Drupal\lightning_core\Element;
+
+/**
+ * Alerts users of Libraries API dependencies.
+ *
+ * A future version of Lightning will remove Libraries API, so
+ * users must add that dependency to composer.json manually.
+ *
+ * @todo Remove this function when Libraries API is no longer shipped with Lightning.
+ */
+function lightning_requirements() {
+  $requirements = [];
+  $libraries_dependencies = [];
+  $enabled_modules = \Drupal::service('extension.list.module')->getAllInstalledInfo();
+
+  foreach ($enabled_modules as $module => $info) {
+    foreach (['libraries:libraries', 'libraries'] as $needle) {
+      if (in_array($needle, $info['dependencies'], TRUE)) {
+        $libraries_dependencies[$module] = $info['name'];
+      }
+    }
+  }
+
+  if ($enabled_modules) {
```

Comment:
This should be `if ($libraries_dependencies)`. :)

Reply:
doh! Yes it should.